bin/history+

```diff
@@ -1,8 +1,51 @@
 #!/bin/bash
 
 function __history+() {
-  local text="${1:-$(date +%Y-%m-%d)}"
-  find "${HOME}"/.bash_history+ -type f | sort | xargs grep "${text}"
+    local text="${1:-$(date +%Y-%m-%d)}"
+    local B0=$(tput setab 0)    # Set background color to black
+    local B9=$(tput setab 9)    # Reset background color to terminal default
+    local F2=$(tput setaf 2)    # Set foreground color to green
+    local F3=$(tput setaf 3)    # Set foreground color to yellow
+    local F4=$(tput setaf 4)    # Set foreground color to blue
+    local F7=$(tput setaf 7)    # Set foreground color to white
+    local F9=$(tput setaf 9)    # Reset foreground color to terminal default
+    local DASH="--------------------------------------------------"
+    local NOW=$(date +"%d-%m-%Y %H:%M:%S")
+    local TITLE="History+"
+    local SESSION="Session"
+    local PID="PID"
+    local DATE="Date/Time"
+    local COMMAND="Command line"
+    local POWEREDBY="Powered by Mathminds"
+
+    if [ "${1}" == "-h" ]
```

Comment:
I haven't really tested your code, but I guess you've forgot to consume the argument "-h" .
You'd better test for this argument when the function starts, set a flag accordingly and shift, in case this argument was specified.

More over, I would not use "-h", since it is in general associated with "help".
I suggest "--color" and/or "--colour", in order to align with ls and grep.

Reply:
Here, the "-h" means "header".
But, I think it will be better if we use "--color" and/or "-H".